document additional Logstash pipeline settings (#98852)
elastic/kibana: docs/api/logstash-configuration-management/create-logstash.asciidoc

  (Required, string) The pipeline definition.

`settings`::

> (Optional, object) The pipeline settings. Supported settings, represented as object keys, are

Reviewer: ```suggestion
(Optional, object) The pipeline settings, represented as object keys, include the following:
```

>   `pipeline.workers`,

Reviewer: ```suggestion
 * `pipeline.workers`
```

>   `pipeline.batch.size`,

Reviewer: ```suggestion
*  `pipeline.batch.size`
```

>   `pipeline.batch.delay`,

Reviewer: ```suggestion
*  `pipeline.batch.delay`
```

>   `pipeline.ecs_compatibility`,

Reviewer: ```suggestion
*  `pipeline.ecs_compatibility`
```

>   `pipeline.ordered`,

Reviewer: ```suggestion
*  `pipeline.ordered`
```

>   `queue.type`,

Reviewer: ```suggestion
*  `queue.type`
```

>   `queue.max_bytes`, and

Reviewer: ```suggestion
*  `queue.max_bytes`
```

>   `queue.checkpoint.writes`.

Reviewer: ```suggestion
*  `queue.checkpoint.writes`
```
